<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>953</width>
    <height>502</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CPTS 451 - Divide &amp; Conquer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>961</width>
      <height>481</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(240, 240, 240);
font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>MS1 - Seach by Location</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>301</width>
        <height>441</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(246, 246, 246);
font: 10pt &quot;MS Shell Dlg 2&quot;;
</string>
      </property>
      <property name="title">
       <string>State / City</string>
      </property>
      <widget class="QComboBox" name="state_list">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>40</y>
         <width>211</width>
         <height>22</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>40</y>
         <width>31</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>State</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>120</y>
         <width>21</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>City</string>
       </property>
      </widget>
      <widget class="QListWidget" name="city_list">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>90</y>
         <width>211</width>
         <height>331</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="businessGroup">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>0</y>
        <width>641</width>
        <height>441</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(246, 246, 246);
font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="title">
       <string>Businesses</string>
      </property>
      <widget class="QTableWidget" name="business_table">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>601</width>
         <height>381</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <attribute name="horizontalHeaderVisible">
        <bool>true</bool>
       </attribute>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>MS2 - Search by Business</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>371</width>
        <height>301</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(246, 246, 246);
font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="title">
       <string>Search</string>
      </property>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>30</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Business:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>260</y>
         <width>51</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Location:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="b_name">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>30</y>
         <width>181</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="b_city">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>260</y>
         <width>181</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QListWidget" name="businesses">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>60</y>
         <width>311</width>
         <height>192</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>953</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
